app/models/site_config.rb

```diff
@@ -50,6 +50,8 @@ class SiteConfig < RailsSettings::Base
                                          default: ApplicationConfig["COMMUNITY_COPYRIGHT_START_YEAR"] ||
                                            Time.zone.today.year
   field :staff_user_id, type: :integer, default: 1
+  field :experience_low, type: :string, default: ApplicationConfig["EXPERIENCE_LOW"] || "Total Newbies"
+  field :experience_high, type: :string, default: ApplicationConfig["EXPERIENCE_HIGH"] || "Senior Devs"
```

Comment:
I think we don't need to add `ApplicationConfig["EXPERIENCE_LOW"]` AND `ApplicationConfig["EXPERIENCE_HIGH"]` here. I understand how it makes sense to add them at first glance since other entries have them, but those are in place because we were using ENV variables before and this avoided breaking some existing forems. Now we generally want to try to avoid adding new ENV variables in favor of purely `SiteConfig`.

Also, since this is a generalization effort I think most communities would benefit from a generalized default as well. So my suggestion would be leaving the defaults as "Total Newbies" and "Experienced Users"/"Advanced Users". There may be other better defaults so I'm open to hearing other's ideas around options for these defaults but anything along those lines should probably be fine.

Reply:
Makes sense, I'll remove the ENV var defaults for now and wait for direction on what the `experience_high` default should be.